defold-spine/src/comp_spine_model.cpp

```diff
@@ -588,19 +652,10 @@ namespace dmSpine
 
             ++num_active;
 
-            float anim_dt = component.m_UseCursor ? 0.0f : dt;
-            component.m_UseCursor = 0;
-
-            if (component.m_Playing)
-            {
-                spAnimationState_update(component.m_AnimationStateInstance, anim_dt);
-            }
-
+            spAnimationState_update(component.m_AnimationStateInstance, dt);
```

Comment:
Is this behaviour verified with animating a cursor?
I thought we wouldn't update the state's times if we use a cursor?

Reply:
I'll check if it works as expected when I get to the office. Just to clarify on what I understood from our Discord talk:

It's ok to let it advance the frame after setting the cursor, because that's the expected behaviour for one-time sets.

For animating the cursor (which is the edge case anyway), if the user sets the playback rate to 0, they'll get correct behaviour. Otherwise, they might get some jitter as the cursor "runs away" after being set. Maybe we should document this.

The alternative is the previous behaviour where animations would just work and one-time sets would cause a one-frame animation pause. We could do that too, but I feel you can do more with the above approach (like, for example, doing seamless looping of parts of an animation) and still get the animation behaviour if you so wish.

Reply:
Your use case sounds valid, so I think it's a good approach.

Reply:
Animating the cursor seems to work fine as far as I can tell.